### PR TITLE
Exception

### PR DESCRIPTION
**Pull request by Gustry**
@myarjunar It might be good to use this `humanise_exception` function. It will give the name and the traceback of the exception.

### CODE REVIEW

**Comment by Gustry** [comment]
@ismailsunni Feel free to merge. It could be nice if @myarjunar can use it in the minimum need calculator.

**Comment by timlinux** [comment]
Funny we used to have something like this in utilities or somewhere that made a nice message out of an exception. I took a quick look but can't remember where it is - might be worth scratching around for some more.... LGTM for now though....

File: safe/utilities/utilities.py [hunk 1]
```diff
@@ -49,7 +49,6 @@
 def get_error_message(exception, context=None, suggestion=None):
```
Comment (Gustry) [inline safe/utilities/utilities.py]: Yes, @timlinux I think you are talking of that one. But this one is doing too much as I don't want the the Messaging system.

Comment (timlinux) [inline safe/utilities/utilities.py]: Ok no problem thanks.

**Pull request merged**